lib/restforce/middleware/authentication/jwt_bearer.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'jwt'
+
+module Restforce
+  class Middleware::Authentication::JWTBearer < Restforce::Middleware::Authentication
+    def params
+      {
+        grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+        assertion: jwt_bearer_token
+      }
+    end
+
+    private
+
+    def jwt_bearer_token
+      JWT.encode claim_set, private_key, 'RS256'
+    end
+
+    def claim_set
+      {
+          iss: @options[:client_id],
+          sub: @options[:username],
+          aud: @options[:host],
+          iat: Time.now.utc.to_i,
+          exp: 3.minutes.from_now.utc.to_i
```

Comment:
This doesn't work as ActiveSupport, which provides `Integer#minutes`, isn't a dependency of Restforce (and I'd prefer not to add it). Can you do this another way using pure Ruby?

Reply:
Good catch!
I've fixed that up, and this time I tested it with `script/console` rather than a rails console that required the gem 😄 

https://github.com/restforce/restforce/pull/479/files#diff-18cd1d533012fd67a233c8205c222769R26